java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -225,4 +225,36 @@ public interface ListBaseCommands {
      * }</pre>
      */
     CompletableFuture<String[]> rpopCount(String key, long count);
+
+    /**
+     * Inserts specified values at the tail of the <code>list</code>, only if <code>key</code> already
+     * exists and holds a list.
+     *
+     * @see <a href="https://redis.io/commands/rpushx/">redis.io</a> for details.
+     * @param key The key of the list.
+     * @param elements The elements to insert at the tail of the list stored at <code>key</code>.
+     * @return The length of the list after the push operation.
+     * @example
+     *     <pre>{@code
+     * Long listLength = client.rpushx("my_list", new String[] {"value1", "value2"}).get();
+     * assert listLength > 2L;
```

Comment:
Nit: This assumption is not always correct what if the list at "my_list" is empty >= would be better.